packages/office-ui-fabric-react/src/utilities/dragdrop/DragDropHelper.tsx

```diff
@@ -270,7 +275,9 @@ export class DragDropHelper implements IDragDropHelper {
    * clear drag data when mouse up outside of the document
    */
   private _onDocumentMouseUp(event: MouseEvent): void {
-    if (event.target === document.documentElement) {
+    const doc = getDocument();
```

Comment:
Any ways to hoist the call to `getDocument` so that you don't have to call twice here and on line 44?

Reply:
The call to `getDocument()` is a getter so it should be inexpensive to call in-line. I'd like to defer hoisting if that's agreeable with you.

https://github.com/OfficeDev/office-ui-fabric-react/blob/8af04c0dfaf4cf2c3510b9af2bf3b1e5d98328c2/packages/utilities/src/dom/getDocument.ts#L14

Reply:
👍